clients/egui/src/widgets/subscription.rs

```diff
@@ -32,6 +32,7 @@ fn subscription_info(
         Some(info) => match &info.payment_platform {
             Stripe { card_last_4_digits } => draw_stripe(ui, card_last_4_digits),
             GooglePlay { account_state } => draw_google_play(ui, account_state),
+            AppStore { .. } => todo!(),
```

Comment:
I think not showing this for now is fine, but `panic`'ing here is prob not the move, let's create a tracking issue for this and not panic.